extension/docs/CancelRefundPayment.md

```diff
@@ -0,0 +1,59 @@
+**Please see [Integration Guide](IntegrationGuide.md) first before continuing with this document.**
+
+## Cancel or refund a payment
+
+1. Shop updates a payment with [a new transaction](https://docs.commercetools.com/http-api-projects-payments#add-transaction)
+    1. for refund
+        ```
+        {
+          "version": ${currentVersion},
+          "actions": [
+            {
+              "action": "addTransaction",
+              "transaction": {
+                "type": "Refund",
+                "amount": {
+                  "currencyCode": "§{currencyCode}",
+                  "centAmount": ${centAmount}
+                },
+                "state": "Initial"
+              }
+            }
+          ]
+        }
+        ```
+    1. for cancel authorization
+        ```
+        {
+          "version": ${currentVersion},
+          "actions": [
+            {
+              "action": "addTransaction",
+              "transaction": {
+                "type": "CancelAuthorization",
+                "amount": {
+                  "currencyCode": "§{currencyCode}",
+                  "centAmount": ${centAmount}
+                },
+                "state": "Initial"
+              }
+            }
+          ]
+        }
+        ```
+    Both transaction types works the same, because Adyen will always pick the right action for the current transaction.
+    If the transaction is not authorized yet, Adyen will try to cancel.
+    If the transaction is already charged, Adyen will do refund.
+1. Extension module finds the first `Payment.transactions` with a transaction `type='Charge' and state='Success'`. 
```

Comment:
Maybe like this?

Reply:
```suggestion
1. Extension module finds the first `Payment` with a transaction `type='Charge' and state='Success'`. 
```

Reply:
Doesn't make much sense, it finds first `transaction`, not first `Payment`. But I see what you mean, my version is a bit repetitive. I will try to think about sth better.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/58/commits/53d4310c30967ceb8bfce1845aedac5da27ef8d0